amoro-web/src/services/optimize.service.ts

```diff
@@ -22,17 +22,23 @@ export function getOptimizerGroups() {
   return request.get('ams/v1/optimize/optimizerGroups')
 }
 
+export function getOptimizerAction() {
+  return request.get('/ams/v1/optimize/action')
+}
+
 export function getOptimizerTableList(
   params: {
     optimizerGroup: string
     dbSearchInput: string
     tableSearchInput: string
     page: number
     pageSize: number
+    action: string[]
+    sortField: string
```

Comment:
I think we shall need sortField as well as sortOrder for ordering feature.

Anyway Let's postpone this sort feature for now(preserve action) and keep this PR as clean as possible.

Reply:
Do you mean I should keep action and remove sortField?

Reply:
yes

Reply:
If we don't need sortOrder now, I will delete it.